packages/accounts/plugingen/phases/plugin-actions/read-actions.ts

```diff
@@ -9,7 +9,7 @@ export const AccountReadActionsGenPhase: Phase = async (input) => {
   const executionAbiConst = `${contract.name}ExecutionFunctionAbi`;
   const executionAbi = extractExecutionAbi(executionFunctions, contract.abi);
 
-  addImport("viem", { name: "GetFunctionArgs", isType: true });
+  addImport("viem", { name: "EncodeFunctionDataParameters", isType: true });
```

Comment:
was this supposed to be part of this PR? or leftover from the bigger one?

Reply:
yea `GetFunctionArgs` isn't the right thing any more